Frontend/src/pages/client/viewProfileInfo/index.jsx

```diff
@@ -0,0 +1,97 @@
+import React, { useEffect, useState } from 'react';
+import { CheckOutlined, CloseOutlined } from "@ant-design/icons";
+import { getUserProfileInfo } from '../../../services/users';
+import { Form, Layout } from 'antd';
+
+const ViewProfileInfoPage = () => {
+
+    const [user, setUsers] = useState();
+    const [temporaryFullName, setTemporaryFullName] = useState({});
+
+    useEffect(async () => {
+        let result = await getUserProfileInfo();
+
+        setUsers(result);
+        setTemporaryFullName({
+            name: result.name,
+            surname: result.surname
+        });
+    }, []);
+
+    if (user === undefined) {
+        return <>Loading...</>
+    }
+
+    return (
+        <Layout className="viewProfileInfoPage">
+
+            <Layout id="infoBlock">
+                <div className="info">
+                    <div className="infoName">
+                        <p>Full name</p>
+                    </div>
+
+                    <div>
+                        <p>{temporaryFullName?.name + ' ' + temporaryFullName?.surname}</p>
+                    </div>
+                </div>
+
+                <Form
+                >
+                    <div className="info">
+                        <div className="infoName">
+                            <p>Name</p>
+                        </div>
+                        <div>
```

Comment:
Add an empty line between div tags that are on the same level. Also fix this in the whole file